Source/EasyNetQ.Management.Client/ManagementClient.cs

```diff
@@ -601,15 +602,44 @@ public async Task<bool> IsAliveAsync(string vhostName, CancellationToken cancell
         return result.Status == "ok";
     }
 
+    public Task<IReadOnlyList<Consumer>> GetConsumersAsync(CancellationToken cancellationToken = default)
+    {
+        return GetAsync<IReadOnlyList<Consumer>>(Consumers, cancellationToken);
+    }
+
+    public Task<bool> HaveHealthCheckClusterAlarmsAsync(CancellationToken cancellationToken = default)
+    {
+        return GetAsync(Health / "checks" / "alarms", c => c == HttpStatusCode.ServiceUnavailable, c => c == HttpStatusCode.OK, cancellationToken);
+    }
+
+    public Task<bool> HaveHealthCheckLocalAlarmsAsync(CancellationToken cancellationToken = default)
+    {
+        return GetAsync(Health / "checks" / "local-alarms", c => c == HttpStatusCode.ServiceUnavailable, c => c == HttpStatusCode.OK, cancellationToken);
+    }
+
     public void Dispose()
     {
         if (disposeHttpClient)
             httpClient.Dispose();
     }
 
-    public Task<IReadOnlyList<Consumer>> GetConsumersAsync(CancellationToken cancellationToken = default)
+    private async Task<bool> GetAsync(
+        RelativePath path,
+        Func<HttpStatusCode, bool> trueCondition,
+        Func<HttpStatusCode, bool> falseCondition,
+        CancellationToken cancellationToken = default
+    )
     {
-        return GetAsync<IReadOnlyList<Consumer>>(Consumers, cancellationToken);
+        using var request = CreateRequest(HttpMethod.Get, path);
+        using var response = await httpClient.SendAsync(request, cancellationToken).ConfigureAwait(false);
```

Comment:
Consider to use `HttpCompletionOption.ResponseHeadersRead` here.

Reply:
Thanks

https://github.com/EasyNetQ/EasyNetQ.Management.Client/commit/29e9e8f817713a6b3ca7bbe46cd8ad20d3770e87